packages/ui/src/components/_utils/analytics.ts

```diff
@@ -23,12 +25,22 @@ type ComponentSpecificCheckEventProps = {
     Checkbox: { value: string }
 }
 
+type ComponentSpecificSelectEventProps = {
+    Select: { value: string, label: string }
+}
+
 
 
-type AnalyticsClickData<K extends keyof ComponentSpecificClickEventProps> = CommonAnalyticsProps<'click', K> & ComponentSpecificClickEventProps[K]
-type AnalyticsCheckData<K extends keyof ComponentSpecificCheckEventProps> = CommonAnalyticsProps<'check', K> & ComponentSpecificCheckEventProps[K]
+type AnalyticsClickData<K extends keyof ComponentSpecificClickEventProps> = CommonAnalyticsProps<'click', K>
```

Comment:
I guess we'll merge this :)

Since 1 component is 1 event usually....but for now it's ok